tests/controllers/doctor.test.js

```diff
@@ -6,21 +6,27 @@ const db = require("../../models");
 
 const { expect } = chai;
 
-describe("Create doctor", async () => {
+describe("POST doctors", async () => {
+  afterEach(async () => {
+    await db.DoctorPracticeArea.destroy({ truncate: { cascade: true } });
+    await db.WorkingHours.destroy({ truncate: { cascade: true } });
+    await db.PracticeArea.destroy({ truncate: { cascade: true } });
+    await db.Doctor.destroy({ truncate: { cascade: true } });
+  });
   it("Create doctor", async () => {
+    // arrange
     const pa = await db.PracticeArea.create({
-      id: 2,
       name: "Surgeon",
-      created_at: new Date(),
-      updated_at: new Date(),
+      createdAt: new Date(),
```

Comment:
Do you need createdAt and updatedAt here?

Reply:
no